ganttproject-tester/test/net/sourceforge/ganttproject/chart/mouse/ChangeTaskProgressRulerTest.java

```diff
@@ -49,6 +49,32 @@ private static Task createTask(TaskManager taskManager) {
         return result;
     }
 
+    public void testProgressConversionToUnits() {
+        TaskChartModelFacade mockChartModel = EasyMock.createMock(TaskChartModelFacade.class);
```

Comment:
`testProgressConversionToUnits` does not share text fixture with the other tests anymore as Progress class is tested directly without mock objects